src/plugin.js

```diff
@@ -86,10 +86,17 @@ const setupHlsHandlers = function(qualityLevels, hls) {
  * Sets up the event handlers for populating and updating the selectedIndex of the
  * QualityLevelList for Dash source.
  *
+ * @return {Function} cleanup function to restore beforeInitialize on Html5DashJS
  * @function setupDashHandlers
  */
 const setupDashHandlers = function() {
+  let oldBeforeInitialize = videojs.Html5DashJS.beforeInitialize;
```

Comment:
If we want to support multiple players on a page with quality levels, we will have to consider this in the scope of the PR I believe